### PR TITLE
Add new table and slider progress bar

### PR DESCRIPTION
**Pull request by is-ashish**
Following changes has been made with this PR.
1. Add new table style css as per new theme
2. Slider progress bar with slider count value minimum 1.

Please find attached screen shot.
Screenshot from 2020-10-19 17-26-44
Screenshot from 2020-10-19 12-26-27
Screenshot from 2020-10-19 12-26-18
Screenshot from 2020-10-19 17-27-26
Screenshot from 2020-10-19 12-24-30
Screenshot from 2020-10-19 19-03-53

Signed-off-by: ashishjain <ashish.jain@mayadata.io>

### CODE REVIEW

File: scss/product/components/_tables.scss [hunk 1]
```diff
@@ -6,26 +6,42 @@
...
-    vertical-align: bottom;
-    border-bottom: (2 * $table-border-width) solid $table-border-color;
-  }
+  // thead th {
+  //   vertical-align: middle;
+  //   border-bottom: (2 * $table-border-width) solid $table-border-color;
+  //   border-top: (2 * $table-border-width) solid $table-border-color;
+  //   color: $table-head-color;
+  //   background: transparent;
+  // }
```
Comment (Pallavi-PH) [inline scss/product/components/_tables.scss:39]: remove this commented code

File: scss/product/components/_tables.scss [hunk 2]
```diff
@@ -78,21 +95,19 @@
...
-    }
-  }
-}
+// .table-hover {
+//   tbody tr {
+//     @include hover {
+//       background-color: $table-hover-bg;
+//     }
+//   }
+// }
```
Comment (Pallavi-PH) [inline scss/product/components/_tables.scss:104]: remove this

File: scss/product/components/_tables.scss [hunk 3]
```diff
@@ -103,8 +118,8 @@
     }
   }
   .table-avatar-label {
-    color: $link-color;
-    margin-left: 14px;
+    color: $blue-500;
```
Comment (Pallavi-PH) [inline scss/product/components/_tables.scss:121]: get this color from variableb css

File: scss/product/components/_tables.scss [hunk 4]
```diff
@@ -218,4 +233,35 @@
...
+    background: $nested-table-background;
+    box-shadow: $nested-table-shadow;
+    border-radius: 0;
+
+    thead {
+      background: $nested-table-background;
+      th {
+        border-bottom: 0px;
+        font-size: 14px;
+        color: $purple;
```
Comment (Pallavi-PH) [inline scss/product/components/_tables.scss:255]: get this color from variable css

**Comment by Pallavi-PH** [comment]
Screenshot 2020-10-19 at 2 55 05 PM

There are some issues with paginations 
1. numbers are not aligned vertically center
2. numbers hover state should also have circle background
3. The number after per page seems to be larger than other text

**Comment by is-ashish** [comment]
@Pallavi-PH have addressed changes PTAL.

File: scss/product/components/_pagination.scss [hunk 5]
```diff
@@ -19,6 +19,8 @@
     color: $pagination-hover-color;
     text-decoration: none;
     background-color: $pagination-hover-bg;
+    border-radius: 25px;
```
Comment (Pallavi-PH) [inline scss/product/components/_pagination.scss:22]: make this 50%
```suggestion
    border-radius: 50%;
```

File: scss/product/components/_pagination.scss [hunk 6]
```diff
@@ -46,6 +48,8 @@
     z-index: 1;
     color: $pagination-active-color;
     background-color: $pagination-active-bg;
+    border-radius: 25px;
```
Comment (Pallavi-PH) [inline scss/product/components/_pagination.scss:51]: same here

File: scss/core/abstracts/_variables.scss [hunk 7]
```diff
@@ -656,8 +666,8 @@ $navbar-light-toggler-border-color: rgba($black, .1) !default;
 $pagination-padding-y:               4px !default;
 $pagination-padding-x:               8px !default;
 $pagination-padding-y-sm:           .25rem !default;
-$pagination-padding-x-sm:           .5rem !default;
-$pagination-padding-y-lg:           .75rem !default;
+$pagination-padding-x-sm:           .6rem !default;
```
Comment (Pallavi-PH) [inline scss/core/abstracts/_variables.scss:669]: make this 0.7rem

File: scss/product/components/_progress.scss [hunk 8]
```diff
@@ -55,3 +55,27 @@
...
+  appearance: none;
+  width: 30px;
+  top: -6px;
+  height: 25px;
+  position: absolute;
+  background: $custom-slider-thumb-background;
+  color: $custom-slider-thumb-color;
+  padding-top: 5px;
+  text-align: center;
+  border-radius: 25px;
```
Comment (Pallavi-PH) [inline scss/product/components/_progress.scss:69]: Same here make it 50%

**Review by Pallavi-PH — APPROVED** [review]
LGTM

**Pull request merged**